string image_path
---
string status